exercises/ruby-1-a/README.md

```diff
@@ -13,25 +13,26 @@ Items are added to the basket using stock-keeping unit (SKU) codes. The items in
 | Potatoes  | 10    | STAPOT |
```

Comment:
@kytrinyx As we're now using numbers with two decimals everywhere else, should we also be using decimals for the whole numbers too?

```suggestion
| Potatoes  | 10.00    | STAPOT |
```